doc/api/stream.md

```diff
@@ -67,7 +74,7 @@ var server = http.createServer( (req, res) => {
   // the end event tells you that you have entire body
   req.on('end', () => {
     try {
-      var data = JSON.parse(body);
+      const data = JSON.parse(body);
     } catch (er) {
       // uh oh!  bad json!
       res.statusCode = 400;
```

Comment:
I propose we rework this example to use a `Writable` and `pipe`, instead of the `'data'` event. I have seen this snippet of code doing more harm than good - as soon as things starts to get complicated, a user __needs__ pipe and backpressure.